OpenAPIService/PowershellFormatter.cs

```diff
@@ -47,6 +50,23 @@ public override void Visit(OpenApiOperation operation)
         {
             var operationId = operation.OperationId;
 
+            if (operation.Extensions.TryGetValue("x-ms-docs-operation-type",
+                                                  out var value) && value != null)
+            {
+                var operationType = ((OpenApiString)value).Value;
+
+                if (operationType.Equals("action", StringComparison.OrdinalIgnoreCase) ||
```

Comment:
```suggestion
                if ("action".Equals(operationType, StringComparison.OrdinalIgnoreCase) ||
```
this style protects you against operationTypeBeing null

Reply:
Interesting...